api-v3-beta.rst

```diff
@@ -0,0 +1,53 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
+putting the focus on the use cases we already known.
+
+Our new API v3 has some big differences compared to the old one that make it more user-friendly and useful.
+
+These differences includes:
+
+* Authenticated only
+* Token based authentication
+* Read and Write
+* Project's slug resource based
+
+Allowing us to build new good features on top of them, like:
+
+* Import a new project
+* List all *my* projects
+* Trigger a build for a specific version
+* Check build status
+* Manage redirects and environment variables programmatically
```

Comment:
This is not a great listing. It either needs a lot more information, or to be split into examples or links to the docs or something.

Reply:
I'm not really sure if adding a lot of more information will work here. It may make the blog post overloaded, in my opinion.

If linking each bullet to the proper section in our documentation works for you, I'd prefer that way. In that case, the post will still be concrete about "Announcing APIv3" mentioning the new features, and if the reader wants to try them out, their jump into reading its docs.

What do you think?